package-scripts.js

```diff
@@ -307,17 +307,17 @@ module.exports = {
         description: 'Build API docs'
       }
     },
-    updateContributors: {
-      script: 'contributors',
-      description: 'Update list of contributors in package.json'
+    updateAuthors: {
+      script: 'scripts/update-authors.js',
```

Comment:
@outsideris I tested your branch and `npm version patch -m "Release v%s"` failed.
I had to change this line to: `script: 'node scripts/update-authors.js', `

Reply:
@outsideris this line does not work.

Reply:
I see, I missed this comment. 
Sorry, I didn't consider windows env because I added hashbang on `update-authors.js'`. 
I will fix it.

Reply:
Fixed